tests/h264Testing/h264Verifier.js

```diff
@@ -0,0 +1,95 @@
+const Promise = require('bluebird');
+const exec = require('child_process').exec;
+const fs = require('fs');
+require('../../lib/utils/KalturaConfig');
+require('../../lib/utils/KalturaCache');
+

```

Comment:
Please add comments mentioning what this class does (be as generous as possible)